src/lib/style/abstracts/_mixins.scss

```diff
@@ -64,6 +65,25 @@
   }
 }
 
+@mixin sticky-column() {
```

Comment:
Here's the sticky-column mixin.  It will make the column sticky, as well as match the striped rows.

Reply:
Weird table header issue in Firefox:
<img width="315" alt="image" src="https://user-images.githubusercontent.com/17580037/144624300-817b85b7-04cf-45b4-aef3-07adef513cde.png">


Sticky + tables can be frustrating, hopefully we can get this working b/c I like the design.